dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -817,4 +1051,17 @@ public ContentType getContentType() throws DotDataException, DotSecurityExceptio
 	public boolean isVanityUrl() throws DotDataException, DotSecurityException {
 		return getContentType().baseType() == BaseContentType.VANITY_URL;
 	}
-}
\ No newline at end of file
+
+    /**
+     * Determines whether this object belongs to a Key/Value Content Type or not.
+     * 
+     * @return If the object is an instance of Key/Value, returns {@code true}. Otherwise, returns
+     *         {@code false}.
+     * @throws DotDataException An error occurred when retrieving information from the data source.
+     * @throws DotSecurityException
+     */
+	public boolean isKeyValue() throws DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)